src/ripple/consensus/Validations.h

```diff
@@ -323,23 +331,23 @@ class Validations
         if (!isCurrent(parms_, t, val.signTime(), val.seenTime()))
             return AddOutcome::stale;
 
-        auto const& id = val.ledgerID();
+        LedgerID const& id = val.ledgerID();
 
         if (!byLedger_[id].emplace(key, val).second)
             return AddOutcome::repeat;
 
         AddOutcome result = AddOutcome::current;
 
         // Attempt to insert
-        auto ins = current_.emplace(key, val);
+        auto const ins = current_.emplace(key, val);
 
         if (!ins.second)
         {
             // previous validation existed, consider updating
-            auto& oldVal = ins.first->second;
+            Validation& oldVal = ins.first->second;
 
-            auto const oldSeq = oldVal.seq();
-            auto const newSeq = val.seq();
+            std::uint32_t const oldSeq = oldVal.seq();
+            std::uint32_t const newSeq = val.seq();
```

Comment:
Thanks for catching this!  I intended to make seq `std::uint32_t` everywhere and had accidentally left it as a `std::size_t` in `RCLValidation`.